workspaces/vscode-base/Dockerfile

```diff
@@ -27,14 +27,14 @@ RUN find /etc/sudoers.d -type f -not -name README -delete && \
 # Apt installs: Enable manpages with "unminimize", and common utilities.
 USER root
 ARG DEBIAN_FRONTEND=noninteractive
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends \
-    man-db unminimize \
-    gosu curl git htop less nano unzip vim wget zip && \
-    yes | unminimize && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/* && \
-    find /tmp -not -path /tmp -delete
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+        man-db unminimize \
+        gosu curl git htop less nano unzip vim wget zip \
```

Comment:
Interesting, this was reformatted but a similar looking bit in `workspaces/rstudio/Dockerfile` was not.

Reply:
Hmm, probably a bug.

Reply:
Resolved in [f1bf2f7](https://github.com/PrairieLearn/PrairieLearn/pull/11718/commits/f1bf2f7589b65f94da866ddf5dd9739f30377d26).